pkgs/development/python-modules/prowlpy/default.nix

```diff
@@ -0,0 +1,64 @@
+{
+  buildPythonPackage,
+  click,
+  fetchFromGitHub,
+  httpx,
+  lib,
+  loguru,
+  pytest-asyncio,
+  pytest-cov-stub,
+  pytestCheckHook,
+  respx,
+  setuptools,
+  xmltodict,
+}:
+
+buildPythonPackage rec {
+  pname = "prowlpy";
+  version = "1.1.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "OMEGARAZER";
+    repo = "prowlpy";
+    tag = "v${version}";
+    hash = "sha256-AotlO1CTe1jv9nu9drQWXYi445Pl5c+89/ep3i+vWwA=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    httpx
+    xmltodict
+  ]
+  ++ httpx.optional-dependencies.http2;
+
+  optional-dependencies = {
+    cli = [
+      click
+      loguru
+    ];
+  };
+
+  pythonImportsCheck = [ "prowlpy" ];
+
+  nativeCheckInputs = [
+    pytest-asyncio
+    pytest-cov-stub
+    pytestCheckHook
+    respx
+  ]
+  ++ lib.concatLists (lib.attrValues optional-dependencies);
+
+  # tests fail without this
+  pytestFlags = [ "-v" ];
```

Comment:
@mweinelt Any idea why this might be the case?

Reply:
Sigh. This reproduces on a checkout of the upstream repo as well.

Reply:
I'd say this is related to the `mock_logger` in tests/test_cli.py, but I don't want to spend time on such a mess tbh.